samples/SampleApp/Startup.cs

```diff
@@ -105,10 +106,23 @@ public static Task Main(string[] args)
                         listenOptions.UseHttps(StoreName.My, "localhost", allowInvalid: true);
                     });
 
+                    options.ListenAnyIP(basePort + 5, listenOptions =>
+                    {
+                        listenOptions.UseHttps(httpsOptions =>
+                        {
+                            var localhostCert = CertificateLoader.LoadFromStoreCert("localhost", "My", StoreLocation.CurrentUser, allowInvalid: true);
+                            httpsOptions.ServerCertificateSelector = (features, name) =>
+                            {
+                                // TODO: Name check, multiple certs, null names.
```

Comment:
Is this something we actually want to do in this sample?

Reply:
No plans to do this in the sample, only showing where to do it. I'll remove the TODO.